packages/sku/src/services/webpack/config/webpack.config.ts

```diff
@@ -64,6 +66,7 @@ const makeWebpackConfig = ({
   const vocabOptions = getVocabConfig(skuContext);
 
   const resolvedPolyfills = resolvePolyfills(polyfills);
+  const webpackAliases = convertPathAliasesToWebpackAliases(pathAliases);
```

Comment:
Didn't realize this would needed for webpack. I guess it sort of doubles up on `srcPaths` config too. 🤦 

Maybe we back out of the webpack stuff and stick with vite-only for now. Sorry for the back and forth 🙏.